native/CMakeLists.txt

```diff
@@ -2254,6 +2211,15 @@ else()
     )
 endif()
 
+source_group(TREE ${CWD} PREFIX "Source Files" FILES ${COCOS2D_SOURCE_LIST})
```

Comment:
Because the source file paths are different, the modification here is to maintain the previous directory structure